sea-orm-macros/src/derives/column.rs

```diff
@@ -91,17 +91,6 @@ pub fn impl_col_from_str(ident: &Ident, data: &Data) -> syn::Result<TokenStream>
         )
     });
 
-    let entity_name = data_enum
-        .variants
-        .first()
-        .map(|column| {
-            let column_iden = column.ident.clone();
-            quote!(
-                #ident::#column_iden.entity_name().to_string()
-            )
-        })
-        .unwrap();
```

Comment:
Hey @tyt2y3, yeah, this isn't possible considering we can `DeriveColumn` on Enum that is not necessarily a `Entity::Column`.